### DISCUSSION TITLE
What are these red triangles on the left side of the editor?

### DISCUSSION THREAD

**Discussion by pilotmoon** [zed-industries/zed#10941] [Feature Requests]
As per screenshot. Can anyone tell me what these indicate?

![CleanShot 2024-04-24 at 17 51 40@2x](https://github.com/zed-industries/zed/assets/17520/e8a28ea4-a1a3-4f02-862b-bd91e617b230)

**Comment by mikayla-maki** [comment]
That is the git gutter! It indicates code has been deleted in that location since the last commit :)

**Reply by pilotmoon** [reply]
Thank you
